client/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "out",
+    "rootDir": "src",
+    "sourceMap": true
+  },
+  "include": ["src"],
+  "exclude": ["node_modules"]
```

Comment:
This should normally not be necessary.

Reply:
you mean all of that?

Reply:
I think this comment is about `"exclude": ["node_modules"]`